service/src/main/java/bio/terra/workspace/service/job/WorkspaceActivityLogHooks.java

```diff
@@ -0,0 +1,124 @@
+package bio.terra.workspace.service.job;
```

Comment:
This hook isn't related to jobs. How about move to bio.bio.terra.worskpace.common.utils, where MdcHook is?